brdf_fallback.py

```diff
@@ -405,94 +402,115 @@ def brdf_indices_quality_check(avg_data=None):
     return filtered_data
 
 
-def get_sds_doy_dataset(h5_info, sds_name):
+def get_sds_doy_dataset(h5_info, sds_name, window):
```

Comment:
h5info dict when passed to this function was sorted before with key param. I don't see it as sorted by datetime...might be reason why the difference in our results? 
 keys = np.sort([k for k in h5_info])